system/Cookie/Cookie.php

```diff
@@ -197,25 +197,23 @@ public static function fromHeaderString(string $cookie, bool $raw = false)
 			$data[strtolower($attr)] = $val;
 		}
 
-		return static::create($name, $value, $data);
+		return new static($name, $value, $data);
 	}
 
 	/**
-	 * Create Cookie objects on the fly.
+	 * Construct a new Cookie instance.
 	 *
-	 * @param string               $name
-	 * @param string               $value
-	 * @param array<string, mixed> $options
+	 * @param string $name    The cookie's name
+	 * @param string $value   The cookie's value
+	 * @param array  $options The cookie's options
 	 *
 	 * @throws CookieException
-	 *
-	 * @return static
 	 */
-	public static function create(string $name, string $value = '', array $options = [])
+	final public function __construct(string $name, string $value = '', array $options = [])
 	{
 		$options += self::$defaults;
 
-		$options['expires'] = static::convertExpiresTimestamp($options['expires']);
+		$options['expires'] = $this->convertExpiresTimestamp($options['expires']);
```

Comment:
Any reason why making the method as not static and available only for the object instance? My intention here is before someone instantiates a `Cookie` object and when given a "non-uniform" timestamp, a dev can simply use `Cookie::convertExpiresTimestamp('blah')` before passing this to the constructor in order to lessen errors.

Reply:
Thanks for explaining this one, and everything